.changeset/calm-clocks-add.md

```diff
@@ -0,0 +1,6 @@
+---
+"@latticexyz/world": patch
```

Comment:
```suggestion
"@latticexyz/world": major
```

Reply:
(it's a breaking change for anyone relying on the previous behavior of the `InstalledModules` table)